### PR TITLE
Update docs based on feedback and add missing results directories

### LINKED ISSUE #261: Improve Bracken execution docs
### Description of the bug

As reported by @alexhbnr 

See comment below - could also make sure that it's clear eveywhere you must explicitly turn on kraken if you also want brakcen results 

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by jfy133**
- Closes #261 
- Partly addresses #262 through better documentation
- Turns on Kraken2 when Bracken is activated  (as this required as an upstream step
- Documents a couple of missing directories in `output.md`
- Adds additional credits on README of people who've provided comments/feedback

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,11 +77,16 @@ The nf-core/taxprofiler pipeline comes with documentation about the pipeline [us
...
-We thank the following people for their extensive assistance in the development of this pipeline:
+We thank the following people for their contributions to the development of this pipeline:
 
-[James A. Fellows Yates](https://github.com/jfy133), [Moritz Beber](https://github.com/Midnighter), [Lauri Mesilaakso](https://github.com/ljmesi), [Sofia Stamouli](https://github.com/sofsam), [Maxime Borry](https://github.com/maxibor),[Thomas A. Christensen II](https://github.com/MillironX), [Jianhong Ou](https://github.com/jianhong), [Rafal Stepien](https://github.com/rafalstepien), [Mahwash Jamy](https://github.com/mjamy).
+[Lauri Mesilaakso](https://github.com/ljmesi), [Tanja Normark](https://github.com/talnor), [Maxime Borry](https://github.com/maxibor),[Thomas A. Christensen II](https://github.com/MillironX), [Jianhong Ou](https://github.com/jianhong), [Rafal Stepien](https://github.com/rafalstepien), [Mahwash Jamy](https://github.com/mjamy), and the [nf-core/community](https://nf-co.re/community).
+
+We also are grateful for the feedback and comments from:
+
+- [Alex Hübner](https://github.com/alexhbnr)
+- Lili Andersson-Li
```
Comment (sofstam) [inline README.md:89]:
```suggestion
- [LilyAnderssonLee](https://github.com/LilyAnderssonLee)
```

Comment (sofstam) [reply README.md:89]: Are the credits for the logo missing?

File: docs/output.md [hunk 2]
```diff
@@ -210,7 +217,10 @@ It is used with nf-core/taxprofiler to allow removal of 'host' (e.g. human) or o
 <summary>Output files</summary>
 
 - `minimap2`
-  - `<sample_id>.bam`: Alignment file in BAM format containing both mapped and unmapped reads.
+  - `build/`
+    - `*.mmi2`: minimap2 indicies of reference genome, only if `--save_hostremoval_index` supplied.
```
Comment (sofstam) [inline docs/output.md:221]:
```suggestion
    - `*.mmi2`: minimap2 indices of reference genome, only if `--save_hostremoval_index` supplied.
```

**Review by sofstam — CHANGES_REQUESTED** [review]
Nice with the extra documentation. A few comments :)

File: README.md (hunk 1 above)
Comment (jfy133) [reply README.md]: Oops yes

File: README.md [hunk 3]
```diff
@@ -77,11 +77,16 @@ The nf-core/taxprofiler pipeline comes with documentation about the pipeline [us
...
-We thank the following people for their extensive assistance in the development of this pipeline:
+We thank the following people for their contributions to the development of this pipeline:
 
-[James A. Fellows Yates](https://github.com/jfy133), [Moritz Beber](https://github.com/Midnighter), [Lauri Mesilaakso](https://github.com/ljmesi), [Sofia Stamouli](https://github.com/sofsam), [Maxime Borry](https://github.com/maxibor),[Thomas A. Christensen II](https://github.com/MillironX), [Jianhong Ou](https://github.com/jianhong), [Rafal Stepien](https://github.com/rafalstepien), [Mahwash Jamy](https://github.com/mjamy).
+[Lauri Mesilaakso](https://github.com/ljmesi), [Tanja Normark](https://github.com/talnor), [Maxime Borry](https://github.com/maxibor),[Thomas A. Christensen II](https://github.com/MillironX), [Jianhong Ou](https://github.com/jianhong), [Rafal Stepien](https://github.com/rafalstepien), [Mahwash Jamy](https://github.com/mjamy), and the [nf-core/community](https://nf-co.re/community).
+
+We also are grateful for the feedback and comments from:
+
+- [Alex Hübner](https://github.com/alexhbnr)
+- [LilyAnderssonLee](https://github.com/LilyAnderssonLee)
```
Comment (jfy133) [inline README.md:89]:
```suggestion
- [LilyAnderssonLee](https://github.com/LilyAnderssonLee)

Credit and thanks also goes to [Zandra Fagernäs](https://github.com/ZandraFagernas) for the logo.
```

**Review by sofstam — APPROVED** [review]

**Pull request merged**